src/VoerroTagsInput.vue

```diff
@@ -400,17 +407,19 @@ export default {
                         value: text,
                     };
 
+                    const queryTerm = this.latinizeTags ? latinize(newTag[this.textField]) : newTag[this.textField];
```

Comment:
I don't think we need to remove diacritics here when checking for whether a tag has been added already. As I mentioned above, there could be two words "Epica" and "Épica" with completely different meanings and because of this code here we won't be able to add them both.